woocommerce/woocommerce
# Strip HTML tags from aria-label in wc_help_tip function

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





When using HTML in `wc_help_tip`, the HTML was being added wrongfully to `aria-label`, and being read by screen readers. This fixes it.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

No need to test, as I've already tested and written unit tests for it.

Alternatively:

1. Install the Product Bundles extension
2. Go to Products > Add New
3. Select "Product bundle" product type
4. Select "Shipping" tab
5. Inspect the ? help label for the bundle types.
6. The `aria-label` attribute should not contain any HTML.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
